sources/app/djam/djam.dylan

```diff
@@ -70,13 +70,17 @@ define function main(name, arguments)
 
     let targets = make(<stretchy-vector>);
     let force? = #f;
+    let jobs = 1;
```

Comment:
Would be sensical to default to max(1, num-cores - 1) or some such?  Just an idea, rather than having to remember to pass --jobs=n